tests/foreman/cli/test_os.py

```diff
@@ -15,12 +15,12 @@
 
 
 POSITIVE_CREATE_DATA = (
-    {'name': gen_string("latin1", 10)},
```

Comment:
i am also going to go through the cli tests at some point and clean up a lot of the other mess.